BackendApartmentReservation/BackendApartmentReservation/Controllers/ApartmentsController.cs

```diff
@@ -9,11 +11,19 @@ namespace BackendApartmentReservation.Controllers
     [ApiController]
     public class ApartmentsController : ControllerBase
     {
+        private ICustomLogger<ApartmentsController> _logger;
+
+        public ApartmentsController(ICustomLogger<ApartmentsController> logger)
+        {
+            _logger = logger;
+        }
+
         [HttpGet]
         [Route("apartments")]
         
         public async Task<IEnumerable<string>> Get()
         {
+            _logger.Info("Initial step");

```

Comment:
## Suggestion
Maybe logging should give info about which component is active? For example 
``` _logger.Info("ApartmentsController - Initial step"); ``` or something similar.

Reply:
This is already taken care of behind the scene. Thats why you dont have to worry about any meta-information